packages/usa-prose/src/styles/_usa-prose.scss

```diff
@@ -1,12 +1,84 @@
 @use "uswds-core" as *;
 
+@mixin prose-test {
+  .usa-prose {
+    & > {
+
+      p {
+        @include typeset-p-base;
+      }
+
+      h1 {
+        margin-bottom: 0;
+        margin-top: 0;
+        clear: both;
+      }
+
+      * + * {
+        margin-top: $theme-paragraph-margin-top !important;
+      }
+
+      * + h1 {
+        margin-top: $theme-heading-margin-top !important;
+      }
+    }
+  }
+}
+
 .usa-prose {
   @include typeset($theme-prose-font-family);
 
   & > {
-    @include usa-paragraph-style;
-    @include usa-headings-styles;
     @include usa-list-styles;
     @include usa-table-styles;
+
+    p {
+      @include typeset-p-base;
+    }
+
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
+      @include typeset-heading-base;
```

Comment:
Adding notes from our call:

* The `!important` flag under the heading selectors was necessary only to override the `!important` flag in the owl(`* + *`) selector.
* The `!important` flag in the owl selector was necessary because of the `@include u-margin-y(0)` mixin within `typeset-p-base`
* By moving the margin reset out of the newly created `typeset-p-base` and into `typeset-p`, we are able to avoid using either `!important` flag
* Without the margin reset, it did become necessary zero out the bottom margin within the owl selector.

Resolved in [7996dfb](https://github.com/uswds/uswds/pull/5158/commits/7996dfbc243d20b55fb8a528ee90aa57d2452c99)